cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -2,6 +2,7 @@
 
 set -e
 
+KUBEVIRT_PROVIDER_EXTRA_ARGS="${KUBEVIRT_PROVIDER_EXTRA_ARGS} --enable-ceph"
```

Comment:
ceph should only be turned on for the k8s provider, AFAIC neither open shift provider has it enabled. I have an open task to enable ceph for OKD-4.1.0

Also cluster-up/* will be overriden when we update the vendoring, so any changes we make (like making certain storage classes the default storage class, should happen in cluster-sync). Now the ceph enabling NEEDS to happen in cluster-up, which is why I sort of punted and told people to set their environment variable prior to calling make cluster-up to enable it.

If we want to enable it all the time, we should create a PR in kubevirtci to enable it there since it got disabled there.

Reply:
Thanks